lib/ramble/ramble/experimental/uploader.py

```diff
@@ -30,6 +30,13 @@ def perform_upload(self, uri, data):
                 f"{self.__class__} requires %{data} argument.")
         pass
 
+def get_user():
```

Comment:
extreme nit;

can we only call getpass if config_user is not set, to avoid making unnecessary system calls?

Reply:
Done!